src/terraform/mlz/main.tf

```diff
@@ -25,8 +25,7 @@ provider "azurerm" {
   metadata_host   = var.mlz_metadatahost
   tenant_id       = var.mlz_tenantid
```

Comment:
I know the Acceptance Criteria doesn't call for it, an oversight, but is `tenant_id` required anymore if we're not planning on authenticating as a Service Principal? I see the need for `subscription_id` and `metadata_host` but think we can safely remove `tenant_id` now too -- lmk if that assumption is wrong.

Reply:
We should both make a note to test this.  I removed all of the references to mlz_tenantid, but tenant_id is still a variable used in the jumpbox access policies.   But I do think whoever is configuring it can take care of that themselves.